server/init-db.js

```diff
@@ -0,0 +1,24 @@
+import config from './server.config.js'
+import postgresStore from './postgres-store.js'
+import User from './models/user.model.js'
+import { run } from 'jest'
+
+async function init () {
+	await postgresStore.init(config.postgres)
+
+	async function dropEverything () {
+		const result = await postgresStore.client.query(
+			"SELECT tablename FROM pg_tables WHERE schemaname = 'public';"
+		)
+		for (const row of result.rows) {
+			if (row.tablename !== 'spatial_ref_sys') {
+				await postgresStore.client.query(`DROP TABLE IF EXISTS "${row.tablename}" cascade`)
+			}
+		}
+	}
+	await dropEverything()
+	await User.generateTable()
```

Comment:
Hello, so are we okay saying that for this first API implementation, we don't have any data in the User table and that we must insert it manually? (I currently observe the creation of an empty table, with `User.generateTable()`) 

Thanks

Reply:
Yes